dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/programrule/engine/DefaultProgramRuleEntityMapperService.java

```diff
@@ -94,7 +94,7 @@
  */
 @Slf4j
 @RequiredArgsConstructor
-@Service("org.hisp.dhis.tracker.imports.programrule.engine.ProgramRuleEntityMapperService")
+@Service
```

Comment:
I think this is our convention https://github.com/dhis2/wow-backend/blob/master/guides/spring_conventions.md#bean-id